core/src/main/scala/org/scalarelational/model/ColumnLike.scala

```diff
@@ -39,8 +39,7 @@ trait ColumnLike[T] extends SelectExpression[T] {
   def isNull = NullCondition(this, Operator.Is)
   def isNotNull = NullCondition(this, Operator.IsNot)
   def ===(value: T) = DirectCondition(this, Operator.Equal, value)
-  def <>(value: T) = DirectCondition(this, Operator.NotEqual, value)
-  def !=(value: T) = DirectCondition(this, Operator.NotEqual, value)
```

Comment:
I leave it to your discretion whether to remove this or just add the other and make it preferred.